### PR TITLE
feat(to-camel-case): init toCamelCase

### LINKED ISSUE #14: Feature request: `toCamelCase / camelize `
Converts the input text to [camel case](https://en.wikipedia.org/wiki/Camel_case).

**Example usage**

```javascript
import toCamelCase from '@plexis/to-camel-case';

toCamelCase('Cool');
// => 'cool'

toCamelCase('cool mate');
// => 'coolMate'

toCamelCase('Hey how are you today?');
// => 'heyHowAreYouToday'

toCamelCase('PascalCase');
// => 'pascalCase'

toCamelCase('kebab-case');
// => kebabCase

```

**Aliases**

```javascript
import toCamelCase from '@plexis/to-camel-case';
import {toCamelCase, camelize, camelCase} from 'plexis';
```

### PR DESCRIPTION
**Pull request by Mercyfulsin**
My attempt at the toCamelCase feature.
Fairly new to PR's on open source so most of my files were created by referencing other projects like toLower or toSucc.
yarn test passed but failed for plexis > index.js, unsure if this is related to my files? Seems to have been issues with toPred.

Fixes #14

### CODE REVIEW

File: packages/plexis/package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "@plexis/to-pred": "^0.0.12",
     "@plexis/to-succ": "^0.0.12",
     "@plexis/to-title": "^0.0.12",
+    "@plexis/to-camel-case": "^0.0.12",
```
Comment (vorillaz) [inline packages/plexis/package.json]: Can you please move this one to alphabetic order?

File: packages/toCamelCase/package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  "version": "0.0.12",
+  "description": "Convert string to camel case",
+  "keywords": [
+    "camel",
+    "case",
+    "camelcase"
+  ],
+  "author": "Andres Ruiz-Velediaz <aruiz858@gmail.com>",
+  "homepage": "https://github.com/Mercyfulsin/plexis#readme",
+  "license": "ISC",
```
Comment (vorillaz) [inline packages/toCamelCase/package.json]: Hey, the license is wrong as well as some keys for the package. 
You can create the new package using the template from here https://github.com/plexis-js/plexis/tree/master/.package-template

File: packages/plexis/src/index.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 export {default as toPred, default as toPredecessor} from '@plexis/to-pred';
 export {default as toSucc, default as toSuccessor} from '@plexis/to-succ';
 export {default as toTitle, default as titleize} from '@plexis/to-title';
+export {default as toCamel, default as toCamelCase} from '@plexis/to-camel-case';
```
Comment (vorillaz) [inline packages/plexis/src/index.js]: According to the issue you shall export 3 methods `toCamelCase, camelize, camelCase` #14

File: packages/toCamelCase/README.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# `@plexis/toCamelCase`
+
+Convert given string to camel case
```
Comment (vorillaz) [inline packages/toCamelCase/README.md]: There is a tiny typo here.

**Comment by vorillaz** [comment]
Hey, @Mercyfulsin no worries at all. First things first thanks for your efforts. Each time you are adding a new method into the main `plexis` package the test is validating that everything is exported as expected. There is a snapshot testing against the main package.
You can use `yarn test -u` in order to update the snapshot.
More infos can be found here https://jestjs.io/docs/en/snapshot-testing
Also, since I have merged some packages you shall rebase and solve any conflicts.

**Comment by Mercyfulsin** [comment]
I keep getting caught with this error when attempting to update the snapshots.
Any idea? I updated the aliases and changed my package license.
Screenshot_1

**Comment by Mercyfulsin** [comment]
Ah I just saw the yarn bootstrap tip. I did that and attempted it again and got the same error but at a different point in the code?
Screenshot_1

**Pull request merged**